save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/LnkUserNotification.kt

```diff
@@ -0,0 +1,26 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+/**
+ * @property notification
+ * @property user
+ * @property isShow
+ */
+@Entity
+class LnkUserNotification(
+
+    @ManyToOne
+    @JoinColumn(name = "notification_id")
+    var notification: Notification,
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    var user: User,
+
+    var isShow: Boolean,
```

Comment:
do we need it? maybe just remove notification if it's shown?

Reply:
Yes, I agree, I deleted it